Cargo.toml

```diff
@@ -22,7 +22,7 @@ categories = [
 include = [ "Cargo.toml", "vm", "README.md", "LICENSE.md" ]
 license = "Apache-2.0"
 edition = "2021"
-rust-version = "1.76.0" # Attention - Change the MSRV in rust-toolchain and in .circleci/config.yml as well
+rust-version = "1.76.0"
```

Comment:
I kinda liked the comment, I saw it being added here (again): https://github.com/ProvableHQ/snarkVM/pull/25/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R25

Reply:
Was removed due to automated release tooling. Should be added back in via @d0cd 's PR that you linked.

Reply:
yeah, me too, a good reminder